<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>111</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>809</width>
       <height>109</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_Listen">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开启服务</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Port">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Send">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_Recv">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_Send">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>接收</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_intertimerStopSend">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>停止发送</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_intertimerSend">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>定时发送</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_interTime">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_COM">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>0</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_openCOM">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>110</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> 打开串口</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_runInfo">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>140</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Botelv">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_avalibCOM">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>可用串口</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_openCOM_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> 打开串口</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_runInfo_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>151</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeView" name="treeView_bmsCalibration">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>821</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
